Functions.Templates/Templates/TimerTrigger-PowerShell/run.ps1

```diff
@@ -0,0 +1,13 @@
+# Input bindings are passed in via param block.
+param($myTimer)
+
+# Get the current universal time in the default string format
+$currentUTCtime = (Get-Date).ToUniversalTime().ToString()
```

Comment:
The `ToString()` is unnecessary as line 13 implicitly calls ToString()

Reply:
True. Fixed.